cfgov/unprocessed/css/molecules/social-media.less

```diff
@@ -92,6 +92,36 @@
   display: none;
 }
 
+//fix issue with padding/margin and verticle rule appearing to the left of print icon
+html[lang='ar'] .m-social-media {
+  &_heading {
+    padding-right: 0;
+  }
+
+  .m-list_item:last-child {
+    margin-right: unit(15px / @base-font-size-px, rem);
+  }
+
+  &_print {
+    padding-left: 0;
+    border-left: none;
+    padding-right: unit((@grid_gutter-width / 2) / @base-font-size-px, em);
```

Comment:
We should probably consistently use `15px` or `@grid_gutter-width` in the margin and padding in this component (also em/rem). `@grid_gutter-width` = 30px, so the `/ 2` here is actually 15px. Since `15px` is used on line 102 and 116, let's make this `padding-right: unit(15px / @base-font-size-px, rem);`  too.